src/Config/Filter/Type/Background.php

```diff
@@ -20,35 +20,23 @@ final class Background extends FilterAbstract
 {
     public const NAME = 'background';
 
-    /**
-     * @var string
-     */
-    private $color;
+    private ?string $color;
 
-    /**
-     * @var string
-     */
-    private $transparency;
+    private ?string $transparency;
 
-    /**
-     * @var string
-     */
-    private $position;
+    private ?string $position;
 
-    /**
-     * @var Size
-     */
-    private $size;
+    private Size $size;
 
     /**
      * @param string|null $color        background color HEX value
      * @param string|null $transparency possible values 0..100
      * @param string|null $position     position of the input image on the newly created background image. Valid values: topleft, top, topright, left, center, right, bottomleft, bottom, and bottomright
      */
     public function __construct(
-        string $color = null,
-        string $transparency = null,
-        string $position = null,
+        ?string $color,
+        ?string $transparency,
+        ?string $position,
```

Comment:
i think we should keep defaulting the constructor arguments to null, to be consistent with the other config models

Reply:
I removed them because the last parameter is not nullable, I didn't thought about that it could make sense using named parameters.

Reply:
Hmm in 8.1 it will throw a deprecation warning: https://3v4l.org/DKJc7 (I'm creating a PR testing with 8.1)

Reply:
interesting. not sure how they thought named parameters vs this new warning should work...

from the way how we use those config models, does it make sense that the fields are nullable? especially color... i assume that the filter defines the defaults - maybe we could move those to this class, or share constants with the default values to make it explicit? that would not fix the warning of php 8.1 though. 

reordering the parameters might make sense actually, with Size being a different type than the strings users would notice an error if they use the config class themselves and not silently swap strings and get unexpected results.